### PR TITLE
Grades1

### PR DESCRIPTION
**Pull request by krissycaron**
@Thomaswaters05 It works hopefully .. please give feedback :-)

### CODE REVIEW

File: grades.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+var scores = [82, 71, 95, 55, 98, 69, 72, 78, 84, 64, 58, 87];
```
Comment (Thomaswaters05) [inline grades.js:1]: I know I told you before but really great job with indenting. I was not doing that this early in the game!

**Comment by Thomaswaters05** [comment]
Good job 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Student Grades</title>
```
Comment (Thomaswaters05) [inline index.html:4]: Are they advising that you put your script tag here or elsewhere?

Comment (morecallan) [reply index.html:4]: @krissycaron -- you should make this change that your big sib advised

**Pull request merged**